test/test_verify_json_payloads.py

```diff
@@ -8,6 +8,37 @@
 from jsonschema import Draft202012Validator, ValidationError
 import json
 
+from esoh.ingest.bufr.bufresohmsg_py import init_bufrtables_py, \
+    init_oscar_py, \
+    destroy_bufrtables_py, \
+    init_bufr_schema_py
+
+
+@pytest.mark.timeout(300)
+@pytest.mark.parametrize("bufr_file_path",
+                         glob.glob("test/test_data/bufr/*.buf*")
+                         + glob.glob("test/test_data/bufr/data????"))
+def test_verify_json_payload_bufr(bufr_file_path):
+    # Load the schema
+    with open("src/esoh/schemas/e-soh-message-spec.json", "r") as file:
+        e_soh_mqtt_message_schema = json.load(file)
+
+    init_bufrtables_py("")
+    init_oscar_py("./src/esoh/ingest/bufr/oscar/oscar_stations_all.json")
+    init_bufr_schema_py("./src/esoh/schemas/bufr_to_e_soh_message.json")
+    msg_build = ingest_to_pipeline(None, None, "testing", testing=True)
+
+    json_payloads = msg_build._build_messages(bufr_file_path, input_type="bufr")
+    destroy_bufrtables_py()
```

Comment:
Is it necessary to manually call this manually, I think this should be handled automatically. C++ destructors should be called when an object becomes unreachable. Alternatively we have to use content managers, ref [this thread](https://stackoverflow.com/questions/55452762/pybind11-destructor-not-invoked)

Reply:
Yes, in that case the memory is not freed by the constructor.
I replace the pointer with a smart pointer and it automatically deletes.

Reply:
Great, will resolve this then.